intel-lustre-client-server/azuredeploy.json

```diff
@@ -0,0 +1,1228 @@
+			"metadata": {
+				"description": "Lustre clients availability set name is important for grouping clients into deployments that can communicate with each other via RDMA"
+			}
+		},
+		"clientVmNamePrefix": {
+			"type": "string",
+			"defaultValue": "lustre",
+			"metadata": {
+				"description": "Prefix for the virtual machine names"
+			}
+		},
+		"clientVmSize": {
+			"type": "string",
+			"defaultValue": "Standard_D2_v2",
+			"metadata": {
+				"description": "Size of the Lustre client VM"
+			},
+			"allowedValues": [
+				"Standard_D2",
+				"Standard_D3",
+				"Standard_D4",
+				"Standard_D11",
+				"Standard_D12",
+				"Standard_D13",
+				"Standard_D14",
+				"Standard_D2_v2",
+				"Standard_D3_v2",
+				"Standard_D4_v2",
+				"Standard_D5_v2",
+				"Standard_D11_v2",
+				"Standard_D12_v2",
+				"Standard_D13_v2",
+				"Standard_D14_v2",
+				"Standard_D15_v2",
+				"Standard_DS2",
+				"Standard_DS3",
+				"Standard_DS4",
+				"Standard_DS11",
+				"Standard_DS12",
+				"Standard_DS13",
+				"Standard_DS14",
+				"Standard_DS2_v2",
+				"Standard_DS3_v2",
+				"Standard_DS4_v2",
+				"Standard_DS5_v2",
+				"Standard_DS11_v2",
+				"Standard_DS12_v2",
+				"Standard_DS13_v2",
+				"Standard_DS14_v2",
+				"Standard_D11",
+				"Standard_D12",
+				"Standard_D13",
+				"Standard_D14",
+				"Standard_G1",
+				"Standard_G2",
+				"Standard_G3",
+				"Standard_G4",
+				"Standard_G5",
+				"Standard_GS1",
+				"Standard_GS2",
+				"Standard_GS3",
+				"Standard_GS4",
+				"Standard_GS5",
+				"Standard_A2",
+				"Standard_A3",
+				"Standard_A4",
+				"Standard_A5",
+				"Standard_A6",
+				"Standard_A7",
+				"Standard_A8",
+				"Standard_A9",
+				"Standard_A10",
+				"Standard_A11",
+				"Standard_A1",
+				"Standard_D1",
+				"Standard_D1_v2",
+				"Standard_DS1",
+				"Standard_DS1_v2"
+			]
+		},
+		"clientCount": {
+			"type": "int",
+			"defaultValue": 2,
+			"allowedValues": [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50],
+			"metadata": {
+				"description": "Number of Lustre clients"
+			}
+		},
+		"clientFilesystemName": {
+			"type": "string",
+			"defaultValue": "scratch",
+			"metadata": {
+				"description": "Name of the Lustre filesystem exposed by the Lustre MGS node"
+			}
+		},
+		"mgsIpAddress": {
+			"type": "string",
+			"defaultValue": "10.1.0.4",
+			"metadata": {
+				"description": "IP address of the Lustre MGS node"
+			}
+		},
+		"clientStorageAccountType": {
+			"type": "string",
+			"defaultValue": "Standard_LRS",
+			"allowedValues": [
+				"Standard_LRS",
+				"Premium_LRS"
+			],
+			"metadata": {
+				"description": "Storage account type (e.g. Premium_LRS or Standard_LRS). Make sure to select Premium_LRS only when using DS-series or GS-series Virtual Machines."
+			}
+		},
+		"existingVnetResourceGroupNameForClient": {
```

Comment:
Since you're already getting existing VNET name above, isn't this duplicate information?